### PR TITLE
Fix/nonce resync

### PR DESCRIPTION
**Pull request by ceseale**
Address #54

### CODE REVIEW

**Comment by Rizato** [comment]
Cancelled run on 379df3e because a20a14f was right after.

**Comment by ceseale** [comment]
Tested by removing https://github.com/polyswarm/relay/blob/29877d53c28f37fa7605e60321195d7a5fac6a21/src/transaction.rs#L139

File: Cargo.lock [hunk 1]
```diff
@@ -1162,7 +1162,7 @@ dependencies = [
 
 [[package]]
 name = "polyswarm-relay"
-version = "0.1.0"
+version = "1.0.0"
```
Comment (Rizato) [inline Cargo.lock]: This should be changed in cargo toml, not lock.

Comment (Rizato) [reply Cargo.lock]: Nvm. It is changed in toml already. Guess this didn't get included.

**Pull request merged**